src/coreComponents/finiteElement/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+#==================================================================================================
+# Package: finiteElement
+#
+# Contains:
+#  - the implementation of finite element formulations for different types of elements.
+#  - the finite element method kernel interface (FEMKI).
```

Comment:
is it called FEMKI anywhere else?

Reply:
I found `FEMKI` term  in the sphinx documentation (developer guide), but I realize it is only used there, probably only for lighter writing. I'll remove it from the `CMakeList` doc.